### DISCUSSION TITLE
Hide earnings on Game Page

### DISCUSSION THREAD

**Discussion by ghost** [RetroAchievements/RAWeb#764] [Ideas]
A option to hide earnings on the Game Page via checkbox (temporary applies to current Game Page) and by default via Settings Menu (applies to all Game Pages) which can be turned on/off temporary for current Game Page.

- [x] Hide Hardcore earnings
- [x] Hide Softcore earnings

**Comment by SyrianBallaS** [comment]
If this is a per user setting, I think it's a good idea. I don't think it would be if it affected everyone at once.

**Comment by ghost** [comment]
@SyrianBallaS only for current logged-in users (based on their earnings). It makes going for completions easier as the earnings aren't displayed anymore and you can focus on those who aren't unlocked yet, XboxAchievements and PSNTropies has something similar last time i've checked those Sites.

**Comment by Jamiras** [comment]
> A option to hide earnings on the Game Page via checkbox (temporary applies to current Game Page)

#1547 adds support for doing this on the game page itself, and is not persisted.
